samples/manage/azure-automation-automated-export/AutoExport.ps1

```diff
@@ -0,0 +1,280 @@
+﻿# The Array that will hold the list of databases objects.
+$dbs = New-Object System.Collections.ArrayList;
+
+# The Enums that describe what state a database is in.
+Add-Type -TypeDefinition @"
+	public enum DatabaseState
+	{
+		ToCopy,
+		Copying,
+		ToExport,
+		Exporting,
+		ToDrop,
+		Finished
+	}
+"@
+
+# The database and server pairs that will be exported.
+$databaseServerPairs =
```

Comment:
I recommend to put all the variables at the beginning of the script so users of the sample only need to change the top section.